client/me/security-2fa-enable/style.scss

```diff
@@ -13,7 +13,6 @@
 }
 
 .security-2fa-enable__app-options {
-	font-color: var(--color-neutral-0);
```

Comment:
Why do we need to remove this?

Reply:
Because of `property-no-unknown` rule. There is no such thing as `font-color` in CSS or is it?

Reply:
Oh, duh 🤦 

While I wonder if we should just switch to `color`, it's probably better to maintain the trunk styles, so removing this sounds good to me